core/src/main/kotlin/Kotlin/DeclarationLinkResolver.kt

```diff
@@ -6,8 +6,6 @@ import org.jetbrains.kotlin.descriptors.CallableMemberDescriptor
 import org.jetbrains.kotlin.descriptors.DeclarationDescriptor
 import org.jetbrains.kotlin.descriptors.TypeAliasDescriptor
 import org.jetbrains.kotlin.idea.kdoc.resolveKDocLink
-import org.jetbrains.kotlin.resolve.descriptorUtil.isEffectivelyPrivateApi
```

Comment:
This change isn't relevant

Reply:
fair enough, I'll create another pull request with the cleanup changes